<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="NEDZLA-LAPTOP" failures="1" tests="1" name="ClientService" time="7.702" errors="0" timestamp="2024-03-29T16:24:42 CET">
  <testcase classname="com.fithub.services.mealplan.test.suites.ClientServiceTest" name="testGetMealPlan_ValidClientIdIsProvided_ReturnsMealPlan" time="0.283">
    <failure type="org.opentest4j.AssertionFailedError" message="
expected: MealPlanResponse(id=1, clientId=1, modified=2024-03-14T19:13, modifiedBy=1)
 but was: null">
      <![CDATA[org.opentest4j.AssertionFailedError: 
expected: MealPlanResponse(id=1, clientId=1, modified=2024-03-14T19:13, modifiedBy=1)
 but was: null
at com.fithub.services.mealplan.test.suites.ClientServiceTest.testGetMealPlan_ValidClientIdIsProvided_ReturnsMealPlan(ClientServiceTest.java:81)
at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:154)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testGetMealPlan_ValidClientIdIsProvided_ReturnsMealPlan -->
</testsuite> <!-- ClientService -->
